### PR TITLE
Device settings create/modify/get endpoints tests.

### PR DESCRIPTION
**Pull request by ice-myles**

### CODE REVIEW

File: cmd/eskimo-hut/devices_test.go [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+//nolint:paralleltest // We can't parallelize it because we don't want to have any user conflicts in the database.
+func TestEskimoHut_CreateDeviceSettings_Success(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	ctx, cancel := context.WithTimeout(context.Background(), bridge.TestDeadline)
+	defer cancel()
+
+	var reqBody, body string
```
Comment (ice-ares) [inline cmd/eskimo-hut/devices_test.go:29]: Move them as close as possible from where they're used

File: cmd/eskimo-hut/devices_test.go [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+		body, status = bridge.CreateDeviceSettings(ctx, t, userID, deviceID, token, reqBody)
+	})
+	THEN(func() {
+		IT("should have status 201 -> http.StatusCreated", func() {
+			assert.Equal(t, 201, status)
+		})
+		IT("should have body with generated data", func() {
+			var expected users.DeviceSettings
+			require.NoError(t, json.Unmarshal([]byte(reqBody), &expected))
+			bridge.AssertCreatedOrModifiedDeviceSettingsResponseBody(t, &expected, body)
```
Comment (ice-ares) [inline cmd/eskimo-hut/devices_test.go:45]: Should not be 1:1 there, so follow the same pattern as with create users: explicitly define what you set in the req  body and explicitly define what u expect in the response, which is at least what u set in the req body, but not always the same

File: cmd/eskimo-hut/devices_test.go [hunk 3]
```diff
@@ -0,0 +1,355 @@
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (ice-ares) [inline cmd/eskimo-hut/devices_test.go:1]: Split file into 1 separate file for each endpoint

File: cmd/eskimo/devices_test.go [hunk 4]
```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BUSL-1.1
+
```
Comment (ice-ares) [inline cmd/eskimo/devices_test.go:2]: Separate into 1 file for each endpoint

**Review by ice-ares — CHANGES_REQUESTED** [review]
Explicitly define/assert/expect stuff; encapsulation is bad for tests (unless u're using it for side effects)

**Comment by ice-myles** [comment]
Was rebased into the `feature/fixing-tests` branch. Will continue from that branch.

**Pull request closed without merging**